daemons/fenced/fenced_commands.c

```diff
@@ -1636,14 +1636,24 @@ fenced_register_level(xmlNode *msg, char **desc, pcmk__action_result_t *result)
         *desc = crm_strdup_printf("%s[%d]", target, id);
     }
 
+    // Ensure a valid target was specified
```

Comment:
I know that this is what was done before but does it make sense to create target & desc if stonith_level_kind is failing?

Reply:
I don't think it makes sense :) but desc is an output variable that is returned with the level notification, so theoretically it's useful even in failure cases